### ISSUE TITLE
[Bug Report] Accessibility: DateTime picker must be keyboard and VoiceOver reachable

### ISSUE THREAD

**Issue by abhipanda** [ElemeFE/element#13191]
### Element UI version
2.4.9

### OS/Browsers version
MacOS Mohave / Safari

### Vue version
2.5.17

### Reproduction Link
[https://jsfiddle.net/5n1sf9L4/1/](https://jsfiddle.net/5n1sf9L4/1/)

### Steps to reproduce
Try tabbing between in and out of input fields and picking date and time using only keyboard. For voice over try enabling VoiceOver feature in Mac.

### What is Expected?
DateTimePicker should be keyboard and voice over accessible

### What is actually happening?
Three accessibility issues:
1. Focus is totally lost in safari while focus outline doesn't show up in chrome & firefox when the date-picker closed.
2. For VoiceOver users the date-time picker is completely inaccessible.
3. Timepicker should receive focus after selecting date

**Comment by wacky6** [comment]
PR welcome
